recipe/meta.yaml

```diff
@@ -133,7 +139,16 @@ outputs:
         - pytest -vvv --pyargs numpy -k "not ({{ tests_to_skip }})" --durations=0
       imports:
         - numpy
+        - numpy.core.multiarray
+        - numpy.core._multiarray_tests
+        - numpy.core.numeric
+        - numpy.core._operand_flag_tests
+        - numpy.core._struct_ufunc_tests
+        - numpy.core._rational_tests
+        - numpy.core.umath
+        - numpy.core._umath_tests
```

Comment:
I don't think we need to test import private modules (i.e., anything starting with `_`);  those aren't particularly guaranteed to stay around nor are they meant for most users.

Reply:
Sure. It's my bad. I'll fix it

Reply:
@chenghlee Fixed